### PR TITLE
Remove unused JS props

### PR DESCRIPTION
**Pull request by externl**
`Ice.PrintStackTraces` and `Ice.IPv6`

### CODE REVIEW

File: js/src/Ice/InstanceExtensions.js [hunk 1]
```diff
@@ -37,9 +37,6 @@ const { TCPEndpointType, WSEndpointType, SSLEndpointType, WSSEndpointType } = Ic
 
 import { Debug } from "./Debug.js";
 
-let _oneOfDone = undefined;
-let _printStackTraces = false;
```
Comment (externl) [inline js/src/Ice/InstanceExtensions.js:41]: As far as I can tell these are never read anywhere.

File: js/test/Ice/proxy/Client.ts [hunk 2]
```diff
@@ -950,148 +950,147 @@ export class Client extends TestHelper {
         p2 = communicator.stringToProxy("test:opaque -e 1.1 -t 4 -v CTEyNy4wLjAuMeouAAAQJwAAAAA=");
         test(communicator.proxyToString(p2) === "test -t -e 1.1:ws -h 127.0.0.1 -p 12010 -t 10000");
 
-        if (communicator.getProperties().getPropertyAsInt("Ice.IPv6") === 0) {
```
Comment (externl) [inline js/test/Ice/proxy/Client.ts:953]: We don't support this property in JS

Comment (pepone) [reply js/test/Ice/proxy/Client.ts:953]: Are all this changes related to indentation change, or did you have to fix anything else?

**Review by bernardnormier — APPROVED** [review]

**Review by pepone — APPROVED** [review]

File: js/test/Ice/proxy/Client.ts (hunk 2 above)
Comment (externl) [reply js/test/Ice/proxy/Client.ts:953]: Just indentation

**Pull request merged**